mfr/extensions/jsc3d/export.py

```diff
@@ -42,3 +46,18 @@ def export(self):
                 extension=extension or '',
                 exporter_class='jsc3d',
             )
+
+        except subprocess.TimeoutExpired as err:
+            raise exceptions.SubprocessError(
+                'JSC3D Conversion timed out.',
+                code=504,
+                process='freecad',
+                cmd=str(err.cmd),
+                returncode=52, # This is not re error code returned by the
```

Comment:
This commenting style is far from **Pythonic**. You can move it to before the `raise` as a comment for the `raise` statement.